docs/changes/2625.features.rst

```diff
@@ -0,0 +1,8 @@
+Add possibility to use ``HIPPARCOS`` catalog to get star positions
```

Comment:
The filename here needs to be `feature` or `api` (better, since this PR contains API changes) not `feature*s*`.

The names need to match one of the pre-defined tags in `pyproject.toml`